src/main/java/net/dv8tion/jda/internal/requests/Route.java

```diff
@@ -202,9 +201,8 @@
         public static final Route ADD_PINNED_MESSAGE =    new Route(PUT,    "channels/{channel_id}/pins/{message_id}",     "channel_id");
         public static final Route REMOVE_PINNED_MESSAGE = new Route(DELETE, "channels/{channel_id}/pins/{message_id}",     "channel_id");
 
-        public static final Route ADD_REACTION =             new Route(PUT,    new RateLimit(1, 250),
-                                                                               "channels/{channel_id}/messages/{message_id}/reactions/{reaction_code}/@me",       "channel_id");
-        public static final Route REMOVE_REACTION =          new Route(DELETE, "channels/{channel_id}/messages/{message_id}/reactions/{reaction_code}/{user_id}", "channel_id");
+        public static final Route ADD_REACTION =             new ReactionRoute(PUT);
```

Comment:
I'm fine with this for now, but if this starts to become something that we notice (routes having route-buckets that are different from their full route) then we need to look at adding additional functionality to the `Route` object to support that usecase.

Don't want or need a bunch of Route subclasses.

Reply:
If this becomes problematic I will change the handling for this but for now this workaround should be sufficient